pandas-stubs/core/construction.pyi

```diff
@@ -22,12 +15,3 @@ def sanitize_array(
     data, index, dtype=..., copy: bool = ..., raise_cast_failure: bool = ...
 ): ...
 def is_empty_data(data) -> bool: ...
-def create_series_with_explicit_dtype(
```

Comment:
drive-by: this was removed from pandas in https://github.com/pandas-dev/pandas/pull/49342/files